linux/keyman-config/keyman_config/view_installed.py

```diff
@@ -41,11 +43,13 @@ def __init__(self, kmp, window):
 
         img_expand = "/usr/share/keyman/icons/expand20.png"
         if not os.path.isfile(img_expand):
-            img_expand = "keyman_config/icons/expand20.png"
-        if not os.path.isfile(img_expand):
-            img_expand = "expand20.png"
-        if not os.path.isfile(img_expand):
-            img_expand = "icons/expand20.png"
+            img_expand = "/usr/local/share/keyman/icons/expand20.png"
```

Comment:
This pattern of looking for an image resource is repeated -- can we refactor that into a function?

Reply:
good point, done

also fixed keyboard icon display in km-config by resizing them on load